ui/desktop/src/main.ts

```diff
@@ -207,7 +213,6 @@ const createTray = () => {
   const tray = new Tray(iconPath);
 
   const contextMenu = Menu.buildFromTemplate([
-    { label: 'Show Window', click: showWindow },
```

Comment:
Why are we dropping this option. Wouldn't the remaining context menu just have a separator and then the `Quit` item?

Reply:
yeah - but I don't think it did anything? (it was a relic from wing to wing I think?) - but could check that out again